docs/learn/voting-dapp/voting-dapp.mdx

```diff
@@ -0,0 +1,679 @@
+Blockchain and web3 have changed our understanding of transparency, trust and decentralization. One of the most incredible use-case for this technology is the development of web3 dApps(Decentralized Applications) that offers solutions that are secure, verifiable and tamper-proof solutions to our everyday challenges. Among all the use-cases, decentralized voting stands out. Decentralized technologies remove middlemen and ensure transparent vote counts, this voting dApp has the potential to change governance system while enhancing the democratic process.
+
+In this tutorial, we will build a simple voting dapp on linea, a zkEVM layer 2 blockchain solution. You will learn how to write, deploy and interact with a smart contract which is designed for decentralized voting process. By the end of this tutorial, you’ll have the solid understanding of the tools and features provided by linea for decentralized applications. 
+
+
+
+## Architecture
+
+The voting dApp has three main components: smart contracts, smart contracts integration and the frontend interface.
+
+The smart contract is one of the main component of the application. This can handle critical functionalities such as registering candidates, allowing users to give their votes based on candidates index numbers, enforcing a one vote per wallet address policy and managing voting period by automatically closing the voting period after a specified duration. The smart contract makes sure that the voting process is secure, transparent and tamper-proof by recording all votes on the blockchain.
```

Comment:
```suggestion
The smart contract is one of the main components of the application. This can handle critical functionalities such as registering candidates, allowing users to give their votes based on candidates index numbers, enforcing a one-vote-per-wallet-address policy and managing voting period by automatically closing it after a specified duration. The smart contract makes sure that the voting process is secure, transparent, and tamper-proof by recording all votes on the blockchain.
```

Above in line 9 you referenced smart contract**s**, in the plural, but here we talk. about a single one -- which is it?